website_v2/gcloud-powershell-website-v2/www/static/app/referenceController.js

```diff
@@ -4,6 +4,7 @@
 app.controller('ReferenceController', ['$scope', '$routeParams',
     function($scope, $routeParams) {
         var prodInfo = $scope.productInfo;
+        if (prodInfo === null) return;
```

Comment:
Edit: There's still an error if you don't load the json file first. Looking into this.